qa/L0_multiple_ports/test.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+# Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+MULTI_PORT_TESTS_PY=multi_port_tests.py
+
+CLIENT_LOG="./client.log"
+
+DATADIR=`pwd`/models
+SP_ARR=(8008 8009 8010 8011 -1 8004 8004 8004 8004 -1 -1)
+HP_ARR=(8008 8010 8010 8009 8005 -1 8005 8005 -1 8004 -1)
+PP_ARR=(8008 8011 8011 8010 8005 8005 -1 8006 -1 -1 -1)
+IP_ARR=(8008 8011 8008 8008 8006 8006 8006 -1 -1 -1 8004)
+SERVER=/opt/tensorrtserver/bin/trtserver
+len=${#SP_ARR[@]}
+
+for (( n=0; n<$len; n++ ))
+do
+:
+  SERVER_ARGS_ADD_GRPC="--grpc-status-port ${SP_ARR[n]} --grpc-health-port ${HP_ARR[n]} \
+    --grpc-profile-port ${PP_ARR[n]} --grpc-infer-port ${IP_ARR[n]} --allow-http 0"
+  SERVER_ARGS_ADD_HTTP="--http-status-port ${SP_ARR[n]} --http-health-port ${HP_ARR[n]} --http-profile-port ${PP_ARR[n]}\
+    --http-infer-port ${IP_ARR[n]} --allow-grpc 0"
```

Comment:
Since you are disallowing both HTTP and GRPC isn't the server going to fail to start? I think you must not have all your changes in here to test all the use-cases we discussed.

Reply:
I have yet to add more test cases. Also I was testing HTTP +/ GRPC Individually hence the above. I can combine the tests